packages/form-plugin/src/directive.ts

```diff
@@ -26,12 +27,24 @@ export class FormDirective implements OnInit, OnDestroy {
   private _updating = false;
 
   constructor(
+    private _actions$: Actions,
     private _store: Store,
     private _formGroupDirective: FormGroupDirective,
     private _cd: ChangeDetectorRef
   ) {}
 
   ngOnInit() {
+    this._actions$
+      .pipe(
+        ofActionDispatched(ResetForm),
+        filter((action: ResetForm) => action.payload.path === this.path),
+        takeUntil(this._destroy$)
+      )
+      .subscribe(({ payload: { value } }: ResetForm) => {
+        this.form.reset(value);
```

Comment:
Should we be using `form.reset(value)` here or `form.resetForm(value)`, which will reset the validations too?

Reply:
We should use form.reset because If we don't use it form values dirty and errors returns previous state. Because directive have a debounce functionality.

Can you check line 98 in directive.ts.

If we use RestAction without form reset
- state will be reset
- observable will spread a value because of the debounce
- UpdateFormValue etc. will be called with form raw value
```typescript
const value = this._formGroupDirective.control.getRawValue();
```